lib/membrane/rtp/parser.ex

```diff
@@ -74,20 +100,13 @@ defmodule Membrane.RTP.Parser do
   end
 
   @impl true
-  def handle_event(:output, %RTCPEvent{} = event, ctx, state) do
-    ctx.pads
-    |> Map.keys()
-    |> Enum.find(fn
-      Pad.ref(:rtcp_output, _id) -> true
-      _pad -> false
-    end)
-    |> case do
+  def handle_event(:output, %RTCPEvent{} = event, _ctx, state) do
```

Comment:
I considered that at first but there is a problem with a `SSRCRouter` receiving events from `StreamReceiveBin`. 

The event will have an `ssrc` param and  the router have pads cached for given `ssrc` to which it forwards the event, the pad gets registered once a packet with a new `ssrc` reaches the router for the first time (the element sending the packets is the RTP parser so the parser's pad gets registered). 

As we want the element to support multiple inputs we would have to perform same registering of `RTCP` parser's pads to which the event would be forwarded (the whole process of caching events until pad is available ect., correct me if I am wrong).

Reply:
If we linked the RTCP pad before the RTP one, I suppose that shouldn't be necessary

Reply:
Can we be sure that some RTCP event  from `RTCP.Parser` having some `ssrc` will arrive before any other RTCP event that will be emitted from `StreamReceiveBin`? If so then pad registering have to be moved from `handle_process` to `handle_event`.

Reply:
Ok, I get it now. I think we can impose some naming convention on the pads ids, so that the corresponding RTCP pad could be named `{:rtcp, rtp_pad_id}`, so we could determine the RTCP pad name from the RTP mapping